File: ZMusic-Plugin/src/main/java/me/zhenxin/zmusic/event/EventBC.java
package me.zhenxin.zmusic.event;

import net.md_5.bungee.api.event.PostLoginEvent;
import net.md_5.bungee.api.plugin.Listener;
import net.md_5.bungee.event.EventHandler;

public class EventBC implements Listener {

    @EventHandler
    public void onPostLoginEvent(PostLoginEvent event) {
        Event.onJoin(event.getPlayer());
    }
}

File: src/main/kotlin/me/zhenxin/zmusic/event/EventBC.kt
package me.zhenxin.zmusic.event

import net.md_5.bungee.api.event.PostLoginEvent
import net.md_5.bungee.api.plugin.Listener
import net.md_5.bungee.event.EventHandler

internal class EventBC : Listener {
    @EventHandler
    fun onPostLoginEvent(event: PostLoginEvent) {
        EventEx.onJoin(event.player)
    }
}
